libs/components/layout/src/lib/modules/action-button/action-button-container.component.ts

```diff
@@ -44,54 +44,66 @@ export class SkyActionButtonContainerComponent
    */
   @Input()
   public set alignItems(value: SkyActionButtonContainerAlignItemsType) {
-    this._alignItems = value;
+    this.#_alignItems = value;
   }
 
   public get alignItems(): SkyActionButtonContainerAlignItemsType {
-    return this._alignItems || 'center';
+    return this.#_alignItems || 'center';
   }
 
   @ContentChildren(SkyActionButtonComponent)
-  private actionButtons: QueryList<SkyActionButtonComponent>;
+  private actionButtons: QueryList<SkyActionButtonComponent> | undefined;
 
   @ViewChild('container', {
     read: ElementRef,
     static: true,
   })
-  private containerRef: ElementRef<any>;
+  private containerRef: ElementRef | undefined;
 
-  private ngUnsubscribe = new Subject<void>();
+  #ngUnsubscribe = new Subject<void>();
 
-  private syncMaxHeightTimeout?: number;
+  #syncMaxHeightTimeout?: number;
 
   private set themeName(value: string) {
```

Comment:
Setters - if applicable - should also use the hash private. We should have no instances of `private ` when we are done with a component.

Reply:
fixed